nemo/collections/asr/models/msdd_models.py

```diff
@@ -982,12 +995,11 @@ def __init__(self, cfg: DictConfig):
         self._init_msdd_model(cfg)
         self.diar_window_length = cfg.diarizer.msdd_model.parameters.diar_window_length
         self.msdd_model.cfg = self.transfer_diar_params_to_model_params(self.msdd_model, cfg)
```

Comment:
why is audio_rttm_map is removed from here?

Reply:
This cannot be defined within the `__init__` as the manifest filepath is not set till `diarize` is called.

It has been moved to here: https://github.com/NVIDIA/NeMo/pull/5945/files/aff83a2532459027ad09c5e68a1b4e17b1801df4#diff-83cbff1d089e98d8c1551c3214d038a32ac03fc3fe6f454e7c0810af149a02d8R1416